### PR TITLE
Usecases fixup

### PR DESCRIPTION
**Pull request by marcoscaceres**
Rewrote a bunch of sections - made a clearer distinction between viewport selection and resolution selection. Added src-n to the list of proposed solutions + description. Resized images, removed scripts at the end of the page that we were not using.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -251,7 +261,7 @@
             multiple versions of an image based on different characteristics
             (e.g., format, resolution, orientation, etc.). This allows the user
             agent to select the optimum image to present to an end-user based
-            on the user agent's environmental conditions, while also providing
+            on the user agent's <a>environmental conditions</a>, while also providing
```
Comment (yoavweiss) [inline index.html]: Shouldn't the `<a>` link somewhere? It's probably some convention I'm not aware of, but I'm not getting its purpose

File: index.html [hunk 2]
```diff
@@ -718,42 +741,18 @@
...
         <p>
-          Some images are best suited to a specific file type for reasons such
-          as file size optimization, alpha transparency, scalability,
-          animation, etc. For example, a photo usually requires good color
-          depth, but does not require alpha transparency or animation; JPEG or
-          WebP are well-suited to these needs and offer good optimization
-          between image quality and file size. Icons are often simpler in terms
-          of color depth, but may require alpha transparency; the PNG and WebP
-          format is better-suited to these needs.
+          Developers rely on different on capabilities of a range of image
```
Comment (yoavweiss) [inline index.html]: s/on//

File: index.html [hunk 3]
```diff
@@ -718,42 +741,18 @@
...
+          formats to communicate effectively. These capabilities include, but
+          are not limited to, alpha transparency, animation, high-color depth
+          support, or particular compressing technique. For example, JPEG
```
Comment (yoavweiss) [inline index.html]: s/particular compressing technique/better compression ratios for a certain image category/ ???

File: index.html [hunk 4]
```diff
@@ -828,6 +815,17 @@
         </li>
         <li>
           <p>
+            The <a>solution</a> MUST support selection based on viewport width
+            and device-pixel-ratio (DPR). Sending the right image for the given
+            dimension and DRP avoids delaying the page load, wasting bandwidth,
```
Comment (yoavweiss) [inline index.html]: s/DRP/DPR/

File: index.html [hunk 5]
```diff
@@ -357,89 +381,89 @@
...
-          browsers with narrow viewports, which are then resized by the browser
-          to fit the design (see, for example, "<a href=
+          size of the viewport. This often results in large images being sent
+          to browsers with narrow viewports, which are then resized by the
+          browser to fit the design (see, for example, <cite><a href=
           "http://blog.netvlies.nl/design-interactie/retina-revolution/">Retina
-          Revolution</a>" by Daan Jobsis).
+          Revolution</a></cite> by Daan Jobsis and the <cite><a href=
+          "http://filamentgroup.com/lab/rwd_img_compression/">compressive
+          images</a></cite> technique). Ideally, developers would like to serve
```
Comment (nwtn) [inline index.html:415]: Compressive images seems like a counter example of when you _wouldn't_ want viewport-based switching. "This often results in large images being sent" implies "large file size", which isn't the case with compressive images. It's in fact using a lack of viewport switching to work.

File: index.html (hunk 4 above)
Comment (marcoscaceres) [inline index.html]: fixed

File: index.html (hunk 5 above)
Comment (marcoscaceres) [inline index.html:415]: true that... will reword

File: index.html (hunk 1 above)
Comment (marcoscaceres) [inline index.html]: ReSpec autolinks these bad boys to the appropriate definition.

File: index.html (hunk 5 above)
Comment (marcoscaceres) [inline index.html:415]: @nwtn - 

> This often results in images with large dimensions (e.g., 2x or more times the size of the viewport) being sent to browsers with narrow viewports

File: index.html (hunk 2 above)
Comment (marcoscaceres) [inline index.html]: fixed

File: index.html (hunk 3 above)
Comment (marcoscaceres) [inline index.html]: used your suggested text

**Comment by marcoscaceres** [comment]
Ok! let's get this puppy published!

**Pull request merged**